### DISCUSSION TITLE
请问如何安装 2.7 并使用 V4 模型

### DISCUSSION THREAD

**Discussion by gh-ray** [PaddlePaddle/PaddleOCR#12453] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubuntu20.04
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：Paddle 2.5.1、PaddleOCR 2.7.0.2
- 运行指令/Command Code：pip install "paddleocr>=2.0.1" # 推荐使用2.0.1+版本
- 完整报错/Complete Error Message：请问如何安装 2.7 版本并使用 V4 模型，安装的时候因为 PyMuPDF 问题一直没有安装成功。
  试了多种版本号，要么版本不符合要求，要么无法安装。
  ERROR: Failed building wheel for PyMuPDF
  ERROR: Could not build wheels for PyMuPDF, which is required to install pyproject.toml-based projects
我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by bltcn** [comment]
请给出具体的报错日志，另外，我可以给你一个我在docker上安装的步骤供你参考
# 自己制作docker
docker pull python:latest
docker run -dp 8868:8868 --name paddleocr:1.0.1 python:latest /bin/bash -c "tail -f /var/log/alternatives.log"
docker exec -it pkuocr /bin/bash

echo "# 默认注释了源码镜像以提高 apt update 速度，如有需要可自行取消注释" > /etc/apt/sources.list
echo "deb https://mirrors.tuna.tsinghua.edu.cn/debian/ bookworm main contrib non-free non-free-firmware" >> /etc/apt/sources.list
echo "# deb-src https://mirrors.tuna.tsinghua.edu.cn/debian/ bookworm main contrib non-free non-free-firmware" >> /etc/apt/sources.list

echo "deb https://mirrors.tuna.tsinghua.edu.cn/debian/ bookworm-updates main contrib non-free non-free-firmware" >> /etc/apt/sources.list
echo "# deb-src https://mirrors.tuna.tsinghua.edu.cn/debian/ bookworm-updates main contrib non-free non-free-firmware" >> /etc/apt/sources.list

echo "deb https://mirrors.tuna.tsinghua.edu.cn/debian/ bookworm-backports main contrib non-free non-free-firmware" >> /etc/apt/sources.list
echo "# deb-src https://mirrors.tuna.tsinghua.edu.cn/debian/ bookworm-backports main contrib non-free non-free-firmware" >> /etc/apt/sources.list

echo "# deb https://mirrors.tuna.tsinghua.edu.cn/debian-security bookworm-security main contrib non-free non-free-firmware" >> /etc/apt/sources.list
echo "# # deb-src https://mirrors.tuna.tsinghua.edu.cn/debian-security bookworm-security main contrib non-free non-free-firmware" >> /etc/apt/sources.list

echo "deb https://security.debian.org/debian-security bookworm-security main contrib non-free non-free-firmware" >> /etc/apt/sources.list
echo "# deb-src https://security.debian.org/debian-security bookworm-security main contrib non-free non-free-firmware" >> /etc/apt/sources.list

apt update && apt install swig libgl1-mesa-glx vim -y
wget https://mirrors.tuna.tsinghua.edu.cn/debian/pool/main/o/openssl/libssl1.1_1.1.1n-0%2Bdeb11u5_amd64.deb
dpkg -i libssl1.1_1.1.1n-0+deb11u5_amd64.deb
rm -rf libssl1.1_1.1.1n-0+deb11u5_amd64.deb

pip install -i https://pypi.tuna.tsinghua.edu.cn/simple "paddleocr>=2.7.0" gradio "paddlepaddle>=2.5.0" fastapi uvicorn
pip install -i https://pypi.tuna.tsinghua.edu.cn/simple prettytable ujson "opencv-python==4.6.0.66" "pillow" "tqdm" "PyYAML>=5.1" "visualdl>=2.2.0" "scipy>=1.0.0" "scikit-learn>=0.21.0" "gast==0.3.3" "faiss-cpu==1.7.4" easydict
pip install -i https://pypi.tuna.tsinghua.edu.cn/simple paddleclas --no-deps
pip install -i https://pypi.tuna.tsinghua.edu.cn/simple nb_log

**Comment by shiyutang** [comment]
可以参考：https://github.com/PaddlePaddle/PaddleOCR/issues/7934
